plot/chart.go

```diff
@@ -339,30 +350,33 @@ func (c *Chart) handleTradingHistoryData(w http.ResponseWriter, r *http.Request)
 	w.Header().Set("Content-Disposition", "attachment;filename=history_"+pair+".csv")
 	w.Header().Set("Transfer-Encoding", "chunked")
 
-	var orders []model.Order
-	if c.paperWallet != nil {
-		orders = c.paperWallet.OrdersByPair(pair)
-	}
+	orders := c.orderStringByPair(pair)
+	log.Println(orders)
```

Comment:
It's a debub?
```suggestion
```

Reply:
Yep it's not finish i will click resolved after finish.

Reply:
Ok, sorry!